llvm/lib/TableGen/TGLexer.cpp

```diff
@@ -38,6 +38,22 @@ struct PreprocessorDir {
 };
 } // end anonymous namespace
 
+/// Returns true if `C` is a valid character in an identifier. If `Position` is
+/// `First`, returns true if `C` is a valid first character of an identifier,
+/// else returns true if `C` is a valid non-first character of an identifier.
+/// Identifiers match the following regular expression:
+///   [a-zA-Z_][0-9a-zA-Z_]*
+enum IDCharPosition {
+  First = 0,
+  NotFirst = 1,
+};
+
+static bool IsValidIDChar(char C, IDCharPosition Position) {
```

Comment:
`functionName` when adding new functions. Some functions don't stick with the convention. Just ignore them.

Reply:
Done.